### PR TITLE
Add conda-forge badge and update install instructions

### PR DESCRIPTION
**Pull request by m-rossi**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Update tests.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `feature`, `ui`, `change`, `documentation`, `breaking`, `ci`
      as they show up in the changelog
- [x] Link this PR to related issues.

As mentioned in #892 here is the update to the documentation. Hope I hit all mentions of conda and got the development version instructions right.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Anaconda-Server Badge](https://anaconda.org/bsteubing/activity-browser/badges/version.svg)](https://anaconda.org/bsteubing/activity-browser)
+[![conda-forge version](https://img.shields.io/conda/vn/conda-forge/activity-browser.svg)](https://anaconda.org/conda-forge/activity-browser)
+[![conda-forge version](https://img.shields.io/conda/vn/bsteubing/activity-browser.svg)](https://anaconda.org/bsteubing/activity-browser)
```
Comment (marc-vdm) [inline README.md:2]: This badge shouldn't be here anymore right?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 [![Anaconda-Server Badge](https://anaconda.org/bsteubing/activity-browser/badges/downloads.svg)](https://anaconda.org/bsteubing/activity-browser)
```
Comment (marc-vdm) [inline README.md:3]: Should be replaced with `[Conda Downloads](https://anaconda.org/conda-forge/activity-browser)`?

**Review by marc-vdm — COMMENTED** [review]
Thanks for working on this.

Two points:
1. It's not clear to me what's now happening with the `mac/ARM` version. This was its own release based on the `dev` version, but it's now gone from `README.md`. Is this supposed to be gone? If so, how can people get access to the `mac/ARM` version?
2. minor comments on badges, see review

[`README.md`](https://github.com/LCA-ActivityBrowser/activity-browser/blob/40c373ace4a914a863ae62a1ed77826976269581/README.md) in current state for reference

**Comment by haasad** [comment]
@marc-vdm I'll try to give some more input sometime over the weekend, can't during the work week. Imo we should wait with updating the installation instructions until we have a) verified that everything works (on all operating systems) and b) have figured out how to manage new releases (stable vs dev etc)

Regarding your concern with the osx-arm: **If** everything works as intended, we can completely drop the workaround for arm, because `bw2calc` from conda-forge is now ["architecture-aware"](https://github.com/conda-forge/bw2calc-feedstock/blob/main/recipe/meta.yaml#L27-L29).

File: README.md (hunk 1 above)
Comment (m-rossi) [reply README.md:2]: Depends, as @haasad said if you want to maybe show a development version you could keep it.

File: README.md (hunk 2 above)
Comment (m-rossi) [reply README.md:3]: Missed that, thanks.

File: README.md (hunk 1 above)
Comment (marc-vdm) [reply README.md]: We show the dev version under the [development version header](https://github.com/LCA-ActivityBrowser/activity-browser/blob/40c373ace4a914a863ae62a1ed77826976269581/README.md#development-version) already, but I don't mind if we'd move that to the top of the file 🤷

**Comment by marc-vdm** [comment]
@haasad Thanks a lot for helping out with this, I'll convert this PR to WIP in a sec so we don't accidentally merge this 😅 . And of course thanks a lot for taking time out of your weekend to help us with this, massively appreciated <3

**Review by Zoophobus — APPROVED** [review]
Thanks to everyone involved for the work and organisation behind this, it's great to have this set up properly

**Pull request merged**